<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>softcluster</class>
 <widget class="QMainWindow" name="softcluster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Soft Cluster</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QTabWidget" name="UI_Tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Create Deformer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Create Cluster Deformer</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QCheckBox" name="UI_Translate_CheckBox">
              <property name="text">
               <string>Translate Deformer</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="+onc" stdset="0">
               <string>'import softcluster;  softcluster.toggle_translate(0)'</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="UI_Rotate_CheckBox">
              <property name="text">
               <string>Rotate Deformer</string>
              </property>
              <property name="+onc" stdset="0">
               <string>'import softcluster;  softcluster.toggle_translate(0)'</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="UI_Spline_CheckBox">
            <property name="text">
             <string>Spline Deformer</string>
            </property>
            <property name="+onc" stdset="0">
             <string>'import softcluster;  softcluster.toggle_spline(0)'</string>
            </property>
            <property name="+ofc" stdset="0">
             <string>'import softcluster;  softcluster.toggle_spline(1)'</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Cluster Guide</string>
            </property>
            <property name="+c" stdset="0">
             <string>'import softcluster;  softcluster.create_guide()'</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>____________________________________________</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Deformer Name</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="UI_cluster_clusterPrefix">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>35</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>35</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>def_</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="UI_cluster_clusterName"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Create Cluster</string>
              </property>
              <property name="+c" stdset="0">
               <string>'import softcluster;  softcluster.create_cluster()'</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="UI_useWeights_CheckBox">
              <property name="text">
               <string>Use Weight Set</string>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Paint Weights</string>
              </property>
              <property name="+c" stdset="0">
               <string>'import softcluster;  softcluster.paint_tool()'</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Weight Sets</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="UI_cluster_list"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="UI_Cluster_AddWeightSet">
                <property name="text">
                 <string>Select</string>
                </property>
                <property name="+c" stdset="0">
                 <string>'import softcluster;  softcluster.select_cluster()'</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="text">
                 <string>Remove</string>
                </property>
                <property name="+c" stdset="0">
                 <string>'import softcluster;  softcluster.use_weight_set()'</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="UI_cluster_refreshWeightSets">
              <property name="text">
               <string>Refresh</string>
              </property>
              <property name="+c" stdset="0">
               <string>'import softcluster;  softcluster.list_clusters()'</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Manage Weights</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Manage Weights</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QListView" name="listView_2">
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Copy Weights</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
